cmd/prometheus-config-reloader/main.go

```diff
@@ -91,6 +97,12 @@ func main() {
 
 	versionutil.RegisterIntoKingpinFlags(app)
 
+	err := web.Validate(*webConfig)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Unable to validate web configuration file. err: "+err.Error())
```

Comment:
`level` outputs all strings to `os.Stdout`, currently. Moving to `os.Stderr` may break this assumption for folks who use these logs in one way or the other (eg., redirecting `>` to a file) but don't actively redirect the latter to the former (`2>&1`), since this has been the behavior throughout the codebase. Nonetheless, even if we decide to direct errors to `os.Stderr`, that should be done for all `level.Error` cases as done above to adhere to the standards being introduced here.

To be explicit, I'm -1 on this since this will break levelled logging for us. Folks can still grep all error logs from `os.Stdout` and redirect those to `os.Stderr` if they want to.

Reply:
To be fair, we're not really consistent: if an error happens before setting up the log struct, it's logged to stderr.

I don't feel strongly about this but I think that we should be consistent and eventually redirect everything to stdout. So for this PR, I'd use `level.Info(log)`.